0-ad-history-mod-package/simulation/templates/units/han/infantry_pikeman_b.xml

```diff
@@ -23,6 +23,11 @@
     <Requirements>
       <Techs datatype="tokens">phase_town</Techs>
     </Requirements>
+    <History>
+        The Ji was a very popular weapon in the Han dynasty. It consisted of two blades. Firstly, the forward blade: it was sharpened on both edges as well as towards the tip—essentially like a sword blade. Secondly, the sideward blade: it was set at a 90-degree angle from the forward blade. This blade typically measured half the length of the forward blade and was sharpened on both edges and towards the tip as well. The two blades of a Ji were forged as one piece and tied to the pole, with the sideward blade running through a hole in the pole to increase stability.
+        The Ji blade itself, as well as the pole it was attached to, came in all different sizes depending on their purpose. The chariots' Ji was held with both hands and had a long pole with only a small blade. The cavalry's Ji was shorter and had a larger blade. Some infantry used a similar one. However, other variations of the Ji for foot soldiers had a much shorter pole and were wielded with only one hand. This left the other hand to carry a shield, or maybe a second Ji. The blade alone could also be used as a short-range weapon, like a dagger. The blade, on its own, was also light enough to be thrown like a knife.
+        The big strength of the Ji blade laid in its versatility. It could, like regular spears, be used for spearing and thrusting. But the forward blade, being sharp on its sides, also allowed for slashing and slicing. The sideward blade could be used for pecking and piercing movements and was also sturdy enough for pushing or pulling. And unlike most spears, pikes, or halberds, the Ji was also very suitable for defense. While the pole and both blades could, of course, block incoming attacks, it was the shape of the blade itself that gave it its remarkable defensive capabilities. In the right angle between the forward and sideward blades, the Ji could catch the enemy's weapon and redirect it.
```

Comment:
```suggestion
        The big strength of the Ji blade laid in its versatility. It could, like regular spears, be used for spearing and thrusting. But the forward blade, being sharp on its sides, also allowed for slashing and slicing. The sideward blade could be used for piercing strikes and was also sturdy enough for pushing or pulling. And unlike most spears, pikes, or halberds, the Ji was also very suitable for defense. While the pole and both blades could, of course, block incoming attacks, it was the shape of the blade itself that gave it its remarkable defensive capabilities. In the right angle between the forward and sideward blades, the Ji could catch the enemy's weapon and redirect it.
```